src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -785,6 +785,7 @@ def aks_create(cmd,     # pylint: disable=too-many-locals,too-many-statements,to
                windows_admin_password=None,
                kubernetes_version='',
                node_vm_size="Standard_DS2_v2",
+               node_osdisk_type="",
```

Comment:
is this required? or optional? if optional pls set it to None

Reply:
optional, will do